### PR TITLE
Add service-level option 'isMediaBadgeEnabled'

### PR DESCRIPTION
**Pull request by santhosh-chinnasamy**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change

- partially addresses #674 
- by default this setting is turned ON

#### Screenshot
image

#### Motivation and Context

- audio icon is displayed every-time when any media(audio/video/gif) is played. This pr makes it service specific so users can turn on/off based on preference.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes

Service level setting for media play indication

### CODE REVIEW

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@cino - could you please chime in whether this should be on by default or the other way around?

**Comment by cino** [comment]
@vraravam I would suggest doing it the other way around as this is a broken feature at the moment. We could change it to on once we have a bullet-proof solution 

@santhosh-chinnasamy Thanks for taking the time to work on this!

**Comment by santhosh-chinnasamy** [comment]
@cino I'll make the changes.

**Comment by santhosh-chinnasamy** [comment]
@vraravam made changes as per @cino's suggestion

File: src/models/Service.ts [hunk 1]
```diff
@@ -59,6 +59,8 @@ export default class Service {
 
   @observable isBadgeEnabled: boolean = true;
 
+  @observable isMediaBadgeEnabled: boolean = false;
```
Comment (vraravam) [inline src/models/Service.ts]: Could the value in this line be obtained from the default settings in the `config.ts` file?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: src/models/Service.ts (hunk 1 above)
Comment (santhosh-chinnasamy) [reply src/models/Service.ts]: yes we can, will do it.

**Review by cino — APPROVED** [review]

**Review by vraravam — APPROVED** [review]

**Pull request merged**